### ISSUE TITLE
Formatting issue with AWS and duplicating folders in postman collections

### ISSUE THREAD

**Issue by willrichboa** [a85/POSTMan-Chrome-Extension#1923]
I've run into an issue where the helperAttributes value in the postman collection is in the wrong format causing issues with command line execution.  

I would expect it to always be in this format:
    "currentHelper": "awsSigV4",
    "helperAttributes": {
                    "accessKey": "a",
                    "secretKey": "b",
                    "region": "c",
                    "saveToRequest": true
                },

_But_
This is how some requests end up being downloaded:

```
            "currentHelper": "awsSigV4",
            "helperAttributes": "{\"accessKey\":\"a\",\"secretKey\":\"b\",\"region\":\"c\",\"saveToRequest\":true}",
```

Steps to reproduce this issue.

Create a new collection.
Create a folder in the collection
Add a request to the collection with an AWS signature.
Duplicate the folder.
Download the collection.
Open the collection json file in a text editor:

In addition the request in the duplicated folder is included in the folders section and in the requests section of the postman_collection file.
Here is an example postman collection.
[!1a.txt](https://github.com/postmanlabs/postman-app-support/files/186531/1a.txt)

**Comment by willrichboa** [comment]
I reported something similar here: https://github.com/postmanlabs/newman/issues/353

**Comment by abhijitkane** [comment]
@willrichboa Thanks for reporting this. W'll be pushing a fix in the next release.

**Comment by abhijitkane** [comment]
@willrichboa Can you confirm this fixed in the latest version of the app (4.2.1)?
